src/components/OpenSeadragonViewer.js

```diff
@@ -359,6 +367,7 @@ export class OpenSeadragonViewer extends Component {
     return (
       <section
         className={classNames(ns('osd-container'), classes.osdContainer)}
+        style={{ cursor: grabbing ? 'grabbing' : 'grab' }}
```

Comment:
Could we push the `grab` default up to the osdContainer styles?

Reply:
Done with this commit: https://github.com/ProjectMirador/mirador/pull/3644/commits/b02ff305ba68c466c91f73e294345754b0e26e53

Reply:
And this commit:
https://github.com/ProjectMirador/mirador/pull/3644/commits/4d79018b9434b78c3828248527f143277b986947